plugins/pdr_deterministic_plugin/tests/simulation_telemetry.py

```diff
@@ -0,0 +1,317 @@
+import time
+from http.server import HTTPServer, BaseHTTPRequestHandler
+from threading import Thread
+from threading import Lock
+import copy
+import argparse
+import random
+from os import _exit
+from os.path import exists
+
+lock = Lock()
+
+PHY_RAW_ERROR_LANE0 = "phy_raw_errors_lane0"
```

Comment:
all errors_lane_i not needed by the PDR anymore, can be removed

Reply:
removed